http/util.ts

```diff
@@ -0,0 +1,9 @@
+// Sleeps the given milliseconds and resolves.
+export function sleep(ms: number): Promise<void> {
```

Comment:
Nice addition. But we may also add this to _std? i often use it personnaly.

Reply:
I agree. This util is very useful. I moved the function to `util/async.ts`. (I also changed the name to `delay` because it sounds more general and node.js has [delay](https://github.com/sindresorhus/delay) module with almost the same purpose.)